packages/palette/src/elements/Select/Select.tsx

```diff
@@ -1,254 +1,246 @@
+    v2: {
+      titleVariant: (variant === "default" ? "text" : "small") as TextVariant,
+      titleTextTransform: null,
+      secondaryTextVariant: "small" as TextVariant,
+    },
+    v3: {
+      titleVariant: "xs" as TextVariant,
+      titleTextTransform: "uppercase",
+      secondaryTextVariant: "xs" as TextVariant,
+    },
+  })
 
   return (
-    <Box width="100%">
-      {title && (
-        <Serif mb="0.5" size="3">
-          {title}
-          {required && <Required>*</Required>}
-        </Serif>
-      )}
-      {description && (
-        <Serif color="black60" mb="1" size="2">
-          {description}
-        </Serif>
-      )}
-      <LargeSelectContainer {...props} p={1}>
-        <select
+    <Box width="100%" {...boxProps}>
+      <Flex
+        as="label"
+        {...(variant === "inline"
+          ? {
+              flexDirection: "row",
+              alignItems: "center",
+            }
+          : {
+              flexDirection: "column",
+              alignItems: "flex-start",
+            })}
+        {...(id ? { for: id } : {})}
+      >
+        <div>
+          {title && (
+            <Text
+              variant={tokens.titleVariant}
+              style={{ textTransform: tokens.titleTextTransform }}
+            >
+              {title}
+              {required && (
+                <Box as="span" color="brand">
+                  *
+                </Box>
+              )}
+            </Text>
+          )}
+
+          {description && (
+            <Text variant={tokens.secondaryTextVariant} color="black60">
+              {description}
+            </Text>
+          )}
+        </div>
+
+        {(title || description) && <Spacer m={0.5} />}
+
+        <Container
+          variant={variant}
           disabled={disabled}
-          name={name}
-          onChange={event => onSelect && onSelect(event.target.value)}
-          value={selected}
+          hover={hover}
+          error={error}
+          focus={focus}
         >
-          {optionTags}
-        </select>
-      </LargeSelectContainer>
+          <select
+            id={id}
+            disabled={disabled}
+            name={name}
+            value={selected}
+            onChange={(event) => {
+              onSelect && onSelect(event.target.value)
+            }}
+            {...selectProps}
+          >
+            {options.map(({ value, text }) => {
+              return (
+                <option value={value} key={value}>
+                  {text}
+                </option>
+              )
+            })}
+          </select>
+        </Container>
+      </Flex>
+
       {error && (
-        <Sans color="red100" mt="1" size="2">
+        <Text variant={tokens.secondaryTextVariant} mt={0.5} color="red100">
           {error}
-        </Sans>
+        </Text>
       )}
     </Box>
   )
 }
 
-/**
- * A small version of drop-down select menu
- */
-export const SelectSmall: SFC<SelectProps> = props => {
```

Comment:
Will want to double check that we're not importing this directly in different places

Reply:
It will still work: at the bottom of the file you'll encounter where I curry them with variant